File: verify2.js
const { chromium } = require('playwright');
(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage({ viewport: { width: 900, height: 1400 } });
  await page.goto('file:///home/admin/coding/morse_trainer/index.html');
  await page.waitForTimeout(500);
  await page.screenshot({ path: 'shot_wide.png' });
  console.log('Screenshot saved');
  
  // Check tree dimensions
  const dims = await page.evaluate(() => {
    const svg = document.getElementById('tree');
    return { w: svg.getAttribute('width'), h: svg.getAttribute('height'), vb: svg.getAttribute('viewBox') };
  });
  console.log('Tree SVG dims:', JSON.stringify(dims));
  
  // Verify timing values
  const timing = await page.evaluate(() => ({
    tC: document.getElementById('tC').textContent,
    tW: document.getElementById('tW').textContent,
    tU: document.getElementById('tU').textContent
  }));
  console.log('Timing display:', JSON.stringify(timing));
  
  await browser.close();
})();

File: verify2.py
from playwright.sync_api import sync_playwright
import json

with sync_playwright() as p:
    browser = p.chromium.launch()
    page = browser.new_page(viewport={"width": 900, "height": 1400})
    page.goto("file:///home/admin/coding/morse_trainer/index.html")
    page.wait_for_timeout(500)
    page.screenshot(path="shot_wide.png")
    print("Screenshot saved")

    dims = page.evaluate("""() => {
        const svg = document.getElementById('tree');
        return { w: svg.getAttribute('width'), h: svg.getAttribute('height'), vb: svg.getAttribute('viewBox') };
    }""")
    print("Tree SVG dims:", json.dumps(dims))

    timing = page.evaluate("""() => ({
        tC: document.getElementById('tC').textContent,
        tW: document.getElementById('tW').textContent,
        tU: document.getElementById('tU').textContent,
        tT: document.getElementById('tT').textContent
    })""")
    print("Timing display:", json.dumps(timing))

    browser.close()
